bucket/slack.json

```diff
@@ -20,11 +21,7 @@
             "Slack"
         ]
     ],
-    "checkver": {
-        "url": "https://downloads.slack-edge.com/releases_x64/RELEASES",
-        "re": "slack-([\\d.]+)-full.nupkg",
-        "reverse": true
-    },
+    "checkver": "Version ([\\d.]+)",
```

Comment:
I would rather see getting it from https://slack.com/intl/en/release-notes/windows with `Slack ([\d.]+)`

Reply:
Ok.